CHANGES.186

```diff
@@ -35,6 +35,7 @@ Fixes:
  * Fix a potential crash in @stats/tables [SW]
  * controls(<obj>, <victim>/<attr>) would return 1 even if <obj> didn't control <victim>. [MG]
  * Although json(object) would correctly return {}, an empty JSON object, other JSON functions would throw an error and say it was invalid. [MG]
+ * Fixed channel_send() incorrectly setting speaker to NOTHING for non-spoofed chat messages when adding text to the channel buffer [Qon]
```

Comment:
Names in square brackets in changes have special meaning, these should just be 'Patch by Qon'